migration/src/m0000345_create_version_comparison_fns/mod.rs

```diff
@@ -0,0 +1,109 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .get_connection()
+            .execute_unprepared(include_str!("is_numeric.sql"))
```

Comment:
I was gonna suggest we introduce a `sql/` dir somewhere to hold all these and then noticed you created a module dir to contain them. That's good, too.